application/pgpool/notes.md

```diff
@@ -0,0 +1,19 @@
+# Notes
+
+## Backlog
+
+```
+BACKEND Error: "Unknown function: pg_catalog.has_function_privilege('crate', 'pgpool_regclass(cstring)', 'execute')"
+```
+
+```
+WARNING:  is_temp_table: unexpected PostgreSQL version: CrateDB 6.2.0-SNAPSHOT (built 9e707bd/NA, Linux 6.12.54-linuxkit amd64, OpenJDK 64-Bit Server VM 25.0.1+8-LTS)
+```
+
+How does pgpool-II find the primary node?
+https://www.pgpool.net/mediawiki/index.php/FAQ#How_does_pgpool-II_find_the_primary_node?
+```
+LOG:  get_query_result: do_query failed
+LOG:  get_query_result: no result returned
+ERROR   : Unknown function: pg_catalog.pg_is_in_recovery()
+```
```

Comment:
Substitute and/or implement `pg_catalog.pg_is_in_recovery()`, i.e. also report to crate/crate?